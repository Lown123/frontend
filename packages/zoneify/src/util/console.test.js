const { divide, header } = require('./console')

describe('console', () => {
  it('should log a divide', () => {
    expect(divide(5)).toMatchSnapshot()
  })

  it('should log a header', () => {
    expect(header('Header')).toMatchSnapshot()
  })
})
