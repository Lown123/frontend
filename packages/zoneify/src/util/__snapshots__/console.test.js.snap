// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`console should log a divide 1`] = `"[2m----[22m"`;

exports[`console should log a header 1`] = `
"[1mHeader[22m
[2m------[22m"
`;
